src/components/DummyStepComponent.tsx

```diff
@@ -0,0 +1,7 @@
+import { StepComponentProps } from "./stepper";
+
+function DummyStepComponent({ title }: StepComponentProps) {
+  return <div data-testid={title}>{title}</div>;
+}
+
+export default DummyStepComponent;
```

Comment:
Are you using this in multiple tests? Might be better to move this definition inside the test file

Reply:
It is also there for manual testing, like the upload images button. I will replace it with the real forms.